Update Traceability Agents table (#479)
Axway/amplify-central: content/en/docs/amplify_relnotes/agent_agentsdk_support_policy.md

Traceability Agents collect usage, metrics, and data plane traffic details and send them to the Amplify platform. In the platform, API consumers and API providers can view the performance and behavior of the assets discovered in the data plane.

### Runtime Compliance Agents

> Runtime Compliance Agents collect date from an third party which captures and scans real-time API traffic to assess security threats.  The APIs are represented as Runtime Unmanaged APIs with a Runtime Compliance score based on the security risk assessment.

Author: ```suggestion
Runtime Compliance Agents collect data from a third party that captures and scans real-time API traffic to assess security threats. The APIs are represented as Runtime Unmanaged APIs with a Runtime Compliance score based on the security risk assessment.
```
